tests/epyccel/test_base.py

```diff
@@ -73,12 +73,12 @@ def test_not(language):
     compare_epyccel(base.not_val, False, language=language)
 
 @pytest.mark.xfail(reason="f2py does not support optional arguments https://github.com/numpy/numpy/issues/4013")
```

Comment:
See function `test_same_string` for an example of how to mark tests as failing in one language only